README.md

```diff
@@ -11,15 +10,63 @@ logic.
     msgid "user"
     msgstr "użytkownika"
 
+With ``AttributiveTranslation`` class you can access ``accusative`` context
+translation through attribute of no-context translation:
+
+    >>> user = AttributiveTranslations(…).gettext('user')
+    >>> user
+    'użytkownik'
+    >>> user.accusative
+    'użytkownika'
+    
+"OK, and what's cool about that?"
+
+Format string syntax introduced in Python 3 allows accessing arguments'
+attributes in format strings. Therefore following is possible:
+
+    >>> 'Wybierz {.accusative} do zmiany'.format(user)
+    'Wybierz użytkownika do zmiany'
+
+We are able to parametrize translation strings with e.g. nouns, which then can
+change grammatical cases in translation. Above example being a gettext
+translation:
+
     msgid "Select {} to change"
     msgstr "Wybierz {.accusative} do zmiany"
+
+Languages that use grammatical cases of nouns are: Armenian, Assamese, most 
+Balto-Slavic languages, Basque, most Caucasian languages, most Dravidian
+languages, German, Icelandic, Japanese, Korean, Latin, Sanskrit, Tibetan, the
+Turkic languages and the Uralic languages.
     
-Second class, ``NoContextFallbackTranslations`` in case of lacking context
-translation makes gettext fall back to translation of message without context.
-In example above if ``msgstr`` for ``user/accusative`` would be empty, it would
- prefer ``"użytkownik"`` over not translated ``user/accusative`` ``"user"``.
+#### Fallback
+
+OK, but let's say we not yet have a context translation:
+
+    msgid "user"
+    msgstr "użytkownik"
+
+    msgctxt "accusative"
+    msgid "user"
+    msgstr ""  # <-- missing translation
+
+Our translation class without fallback will by default fallback accusative of
+user to original English string:
+
+    >>> user = AttributiveTranslations(…).gettext('user')
+    >>> 'Wybierz {.accusative} do zmiany'.format(user)
+    'Wybierz user do zmiany'
+
+``NoContextFallbackTranslations`` class will make us fall back to no-context
+translation which makes a bit better user experience:
```

Comment:
```suggestion
``NoContextFallbackTranslations`` class will fall back to the translated string without context. This may be more desirable in some use cases.
```